.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build Workflow
+
+on:
+  workflow_dispatch:
+  push:
+   branches:
+     - master
+     - develop
+     - 'release/*'
+     - 'feature*'
+     - 'tt-138-create-time-sync-library'
```

Comment:
Is this going to be remove  prior to merge?

Reply:
Let me remove this.